### PR TITLE
Only log if the install mode is 'temporary'

### PR DESCRIPTION
**Pull request by k88hudson**

### CODE REVIEW

File: src/dns-test.js [hunk 1]
```diff
@@ -503,4 +506,13 @@ async function main() {
...
 }
 
-main();
+function onInstalled({temporary}) {
+    // Enable logging only for local debugging
+    loggingEnabled = temporary;
+    logMessage("Logging enabled");
+
+    browser.runtime.onInstalled.removeListener(onInstalled);
+    main();
```
Comment (k88hudson) [inline src/dns-test.js:515]: This does change when `main` runs slightly, we could keep this outside (although then you might get a race condition with logging being not enabled in time or whatever)

**Comment by ekr** [comment]
This PR has a real problem in that it means that the add-on only runs once when installed, not on each startup.

**Comment by k88hudson** [comment]
Oops, you're right... :P

**Comment by k88hudson** [comment]
Alright, I found a better way to read this directly v.s. having to rely on the install event

**Review by ekr — APPROVED** [review]

**Pull request merged**